sqlalchemy_bigquery/base.py

```diff
@@ -686,6 +770,147 @@ def visit_drop_table_comment(self, drop):
         table_name = self.preparer.format_table(drop.element)
         return f"ALTER TABLE {table_name} SET OPTIONS(description=null)"
 
+    def _validate_option_value_type(self, option: str, value):
+        """
+        Validates the type of the given option value against the expected data type.
+
+        Args:
+            option (str): The name of the option to be validated.
+            value: The value of the dialect option whose type is to be checked. The type of this parameter
+                is dynamic and is verified against the expected type in `self.option_datatype_mapping`.
+
+        Returns:
+            bool: True if the type of the value matches the expected type, or if the option is not found in
+                `self.option_datatype_mapping`.
+
+        Raises:
+            TypeError: If the type of the provided value does not match the expected type as defined in
+                `self.option_datatype_mapping`.
+        """
+        if option in self.option_datatype_mapping:
+            self._raise_for_type(
+                option,
+                value,
+                self.option_datatype_mapping[option],
+            )
+
+        return True
+
+    def _raise_for_type(self, option, value, expected_type):
+        if type(value) is not expected_type:
+            raise TypeError(
+                f"bigquery_{option} dialect option accepts only {expected_type},"
+                f" provided {repr(value)}"
+            )
+
+    def _process_time_partitioning(
+        self, table: Table, time_partitioning: TimePartitioning
+    ):
+        """
+        Generates a SQL 'PARTITION BY' clause for partitioning a table by a date or timestamp.
+
+        Parameters:
```

Comment:
Let's change this to Args to be consistent throughout our code (see function `_validate_option_value_type()` above and the [Google style guide](https://google.github.io/styleguide/pyguide.html).

NOTE: I am aware that when this module was originally generated pretty much none of it had docstrings, so I appreciate this effort to ensure the overall quality of any new code.

```suggestion
        Args:
```

Reply:
It makes sense.